### ISSUE TITLE
Relx issue on MacOS

### ISSUE THREAD

**Issue by falood** [erlio/vernemq#366]
### Environment

- VerneMQ Version: 1.0.0
- OS: Mac OS
- Erlang/OTP version (if building from source): 19

### Expected behavior
got such exception when run `make rel`, and I fixed it by using local rebar3 `REBAR ?= ~/.mix/rebar3`.

```Error: function_clause
[{relx,format_error,
       [{error,eacces}],
       [{file,"/home/tristan/Devel/rebar3/_build/default/lib/relx/src/relx.erl"},
        {line,220}]},
 {relx,report_error,2,
       [{file,"/home/tristan/Devel/rebar3/_build/default/lib/relx/src/relx.erl"},
        {line,336}]},
 {rebar_relx,do,4,
             [{file,"/home/tristan/Devel/rebar3/_build/default/lib/rebar/src/rebar_relx.erl"},
              {line,38}]},
 {rebar3_cuttlefish_release,do,1,
                            [{file,"/path_to_vernemq/_build/default/plugins/rebar3_cuttlefish/src/rebar3_cuttlefish_release.erl"},
                             {line,89}]},
 {rebar_core,do,2,
             [{file,"/home/tristan/Devel/rebar3/_build/default/lib/rebar/src/rebar_core.erl"},
              {line,153}]},
 {rebar3,main,1,
         [{file,"/home/tristan/Devel/rebar3/_build/default/lib/rebar/src/rebar3.erl"},
          {line,66}]},
 {escript,run,2,[{file,"escript.erl"},{line,760}]},
 {escript,start,1,[{file,"escript.erl"},{line,277}]}]
```

**Comment by dergraf** [comment]
@falood Thanks for mentioning. 

cc: @tslaughter have you seen this before?

**Comment by dergraf** [comment]
Could our Apple userbase verify this issue? :)

**Comment by ioolkos** [comment]
LOL, that would be me, I guess :)
Should probably update to a modern OS X though

**Comment by codeadict** [comment]
I had the error, is because Rebar can't access `_build` i imagine, i solved by changing all the VerneMQ directory permissions or running with sudo. I don't know the real cause but that made the trick. @falood if you have latest El Capitan, https://github.com/codeadict/homebrew-vernemq can be useful.

**Comment by falood** [comment]
Thanks @codeadict , I have fixed it myself 🙂

**Comment by larshesel** [comment]
This is the same as discussed at the bottom of #206 and looks to be an issue with erlware/relx#564 and our current rebar 3.3.2 version - the bug should be fixed in rebar 3.3.6, but it would be nice if someone could verify. In any case we'll likely try to include rebar 3.3.6 or newer in the next bugfix release.

**Comment by larshesel** [comment]
Guess we can close this.
